docs/ui/roles.md

```diff
@@ -0,0 +1,138 @@
+---
+description: Configure user workspace roles in Prefect Cloud.
+tags:
+    - UI
+    - dashboard
+    - Prefect Cloud
+    - accounts
+    - teams
+    - workspaces
+    - organizations
+    - custom roles
+    - RBAC
+---
+
+# User roles in Prefect Cloud
+
+[Organizations](/ui/organizations/) in Prefect Cloud 2 let you to give people in your organization access to the appropriate Prefect functionality in the specific workspaces. 
+
+Role-based access control (RBAC) functionality in Prefect Cloud enables you to assign users granular permissions to perform certain activities within an organization or a workspace.  
+
+To give users access to functionality beyond the scope of Prefect’s built-in workspace roles, you may also create custom roles for users.
+
+[Contact sales](https://www.prefect.io/pricing) to learn more about annual plans giving you control over organizations and user roles.
+
+## Built-in roles
+
+Assigned roles give users abilities at either the organization level or at the individual workspace level. Organizational roles apply to all workspaces within an organization. Workspace roles apply to only the workspace in which the role is assigned.
```

Comment:
```suggestion
Assigned roles give users abilities at either the organization level or at the individual workspace level. Organizational roles apply to all workspaces within an organization. Workspace roles apply to only the workspace in which the role is assigned.
```

Hmm I think we need to be really careful here between "Organization-wide role" / default role and "your role in an organization".

I think it would help to distinguish towards the start of the doc between workspace roles and organization roles